### PR TITLE
5571migrate tests to junit5

### PR DESCRIPTION
**Pull request by fspirola**
## PR description
issue #5571

### CODE REVIEW

**Comment by fspirola** [comment]
#5571

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (fab-10) [inline .vscode/settings.json:1]: Please exclude IDE files from commits

File: ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/peers/EnodeURLImplTest.java [hunk 2]
```diff
@@ -500,7 +500,7 @@ public void toURI_WithHostnameShouldWorkWhenDnsEnabledAndUpdateEnabled()
         "enode://"
             + VALID_NODE_ID
             + "@"
-            + InetAddress.getLocalHost().getCanonicalHostName()
+            + InetAddress.getLocalHost().getHostName()
```
Comment (fab-10) [inline ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/peers/EnodeURLImplTest.java:503]: Is this related to Junit5 port?

**Review by fab-10 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (fspirola) [reply .vscode/settings.json]: IDE files removed from commit. Include .vscode/ in gitignore

File: ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/peers/EnodeURLImplTest.java (hunk 2 above)
Comment (fspirola) [reply ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/peers/EnodeURLImplTest.java]: No, just a mistake.

**Review by macfarla — APPROVED** [review]
LGTM

**Comment by macfarla** [comment]
this one is superseded by #5571 - right? can this one be closed?

**Comment by fspirola** [comment]
Yes, #5571 can be closed.

**Comment by macfarla** [comment]
closing this one as it's a subset of #5571
